.mergify.yml

```diff
@@ -23,3 +23,25 @@ pull_request_rules:
       queue:
         method: rebase
         name: default
+  - name: Needs landing
+    conditions:
+      - label=Needs Landing
+      - "#approved-reviews-by>=1"
+      - -draft
+      - label!=Work In Progress
+      - label!=Do Not Land
+    actions:
+      queue:
+        method: merge
+        name: default
+  - name: Needs landing - Squash
+    conditions:
+      - label=Needs Landing (Squash)
+      - "#approved-reviews-by>=1"
+      - -draft
+      - label!=work in progress
+      - label!=do not land
+    actions:
+      queue:
+        method: merge
```

Comment:
Should this not be `squash`?

Reply:
You are right... but I think squash does a rebase and we will have the behaviour commented below...

Reply:
From what I understand from [here](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/incorporating-changes-from-a-pull-request/about-pull-request-merges) I don't think that a squash & merge does a rebase, but I could be wrong.